src/Spd.Resource.Repository/ControllingMemberCrcApplication/Mappings.cs

```diff
@@ -0,0 +1,147 @@
+﻿using AutoMapper;
+using Microsoft.Dynamics.CRM;
+using Spd.Resource.Repository.Alias;
+using Spd.Resource.Repository.PersonLicApplication;
+using Spd.Utilities.Dynamics;
+using Spd.Utilities.Shared.Tools;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Spd.Resource.Repository.ControllingMemberCrcApplication;
+internal class Mappings : Profile
+{
+    public Mappings()
+    {
+        //TODO: check where should keep HasBankruptcyHistory and detail
```

Comment:
check dynamics spd_application, I see there is bankrupcy

Reply:
added